style.css

```diff
@@ -180,19 +183,46 @@ body {
   flex-direction: column;
 }
 
-/* for component demo */
-h2 ~ .badge-text-container {
-  width: fit-content;
-  background-color: var(--BLACK);
-  color: var(--OFFWHITE);
-  padding: 0 0.5em;
+/* Grids demo  */
+.grid-wrapper {
+  width: 100%;
+  max-height: 10rem;
+  background-color: var(--GRAY);
+  grid-gap: 2rem;
```

Comment:
use gap instead of grid-gap. It is the older version of gap.

Reply:
Sure.